frontend/src/pages/MedDog.scss

```diff
@@ -6,16 +6,13 @@
   width: 100%;
 }
 
-.title{
+.titleMeddog{
   position: absolute;
   justify-content: center;
```

Comment:
Is this a flexbox or grid? 🧐 position: absolute without positional rules looks suspicious

Reply:
yeah, you're right. still need to fix all css files